### DISCUSSION TITLE
Screen reader accessibility in quarto with observable (OJS cells)Your question in one sentence (e.g., How do I preview documents with Quarto?)

### DISCUSSION THREAD

**Discussion by RichCaloggero** [quarto-dev/quarto-cli#9435] [Q&A]
### Description

Quarto version 1.4.553
windows 10

I'm new here, so sorry if I'm not quite understanding how things work...

I'm a screen reader user and want to create a jupyter notebook experience with quarto. From what I gleaned from the docs for observable (  https://observablehq.com/@observablehq/observable-javascript ), you need to use their site and server to do this? However, quarto using OJS cells runs observable, so is it possible to create notebook like reactive cells using the libraries embedded in quarto and the quarto cli preview server?

**Comment by cscheid** [comment]
Please refer to our documentation on this: https://quarto.org/docs/interactive/ojs/
